build_tag_push.py

```diff
@@ -1,101 +1,130 @@
 # Name: Alyssa Kelley
-# Last worked on: Jan. 8, 2019
+# Last worked on: Jan. 14, 2019
 
 import yaml # Parse the add.yml file
-import sys # Potentially use argv[1] for the GitHub URL we are accessing the add.yml file from...
-import os # Using this to store variables as environment variables and run terminal commands.
+import sys 
+import os
+import subprocess
+from shutil import copyfile # Allowing me to copy dockerfile context to new temp dir
+
+Docker_Hub_Name = "alyssakelley" # This is my personal docker namespace/username for testing.
+# If we want to make this a user input option --> Docker_Hub_Name = input('Enter your Docker namespace: ')
```

Comment:
Yes, I think we want to make this user input but I wouldn't do it as a prompt - I'd make it a command line argument and use `argparse`: https://docs.python.org/3/library/argparse.html

The reason is it's much easier to script that way. For example, we'll want to run this from the CI server (circleCI or TravisCi). Those have mechanisms to securely store "secrets" like your dockerhub username and password and they are made available to your build via environment variables so our script could then look like `python build_tag_push.py --dockerhub_user=$DOCKERHUB_USER --dockerhub_password=$DOCKERHUB_PASS`